lib/tara/installer.rb

```diff
@@ -63,6 +76,50 @@ def bundle_gems
       end
     end
 
+    def find_installed_gems
+      Bundler.with_clean_env do
+        definition = Bundler::Definition.build('lib/vendor/Gemfile', 'lib/vendor/Gemfile.lock', false)
+        @without_groups.each do |group|
+          definition.add_optional_group(group)
+        end
+        specs = definition.specs.map do |gem_spec|
+          spec = {
+            :name => gem_spec.name,
+            :full_name => gem_spec.full_name,
+            :full_gem_path => gem_spec.full_gem_path,
+            :spec_file => gem_spec.spec_file,
+            :bin_dir => gem_spec.bin_dir,
+          }
+          if spec[:full_gem_path].start_with?(Bundler.bundle_path.to_s) # Local gem
+            spec[:relative_path] = Pathname.new(spec[:full_gem_path]).relative_path_from(Bundler.bundle_path).to_s
+          end
+          spec
+        end
+      end
+    rescue Bundler::GemNotFound => e
+      []
+    end
+
+    def copy_local_gems
+      local_gems = find_installed_gems
+      target_directory = File.join(@package_dir, 'lib/vendor', Bundler.ruby_scope)
+
+      Bundler.with_clean_env do
+        spec_dir = File.join(target_directory, 'specifications')
+        bin_dir = File.join(target_directory, 'bin')
+        FileUtils.mkdir_p(spec_dir)
+        FileUtils.mkdir_p(bin_dir)
+        FileUtils.mkdir_p(File.join(target_directory, 'gems'))
+        FileUtils.mkdir_p(File.join(target_directory, 'bundler/gems'))
+
+        local_gems.each do |gemspec|
+          next unless gemspec[:relative_path]
```

Comment:
If we're gonna skip the ones with a `:relative_path` key, I think it would be better to just exclude them in `#find_installed_gems`.